testapi.pm

```diff
@@ -1037,6 +1037,15 @@ sub validate_script_output($&;$) {
     my $output = script_output($script, $wait);
     my $res = 'ok';
 
+    if ($ENV{SANITYCHECK}) {
+        # quick sanity check for check functions:
+        $_ = '';
+        my $ret1 = $code->();
+        $_ = 'foobar';
```

Comment:
I don't think this is worth it. This should actually be possible to check by means of static analysis - but calling the function just so is a no-go. And if you hide it behind a variable, it's even more no-go as we would still do this only by chance.

Reply:
we can certainly drop the if.
But I recently saw http://perlmonks.org/?node_id=663393 which seems to suggest that static analysis is impossible - at least in the general case.

Reply:
I don't care about the general case